.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+branch = True
+source = hotsos
```

Comment:
Probably want to use omit as well, for example:

omit = tests/*

That would probably increase the coverage numbers.

Reply:
done